envs/monkey_zoo/blackbox/test_configurations/zerologon.py

```diff
@@ -15,7 +15,7 @@
 
 def _add_exploiters(agent_configuration: AgentConfiguration) -> AgentConfiguration:
     exploiters: Dict[str, Mapping] = {
-        "Zerologon": {},
+        "Zerologon": {"max_attempts": 2000},
```

Comment:
See https://github.com/guardicore/monkey/pull/3295/files#r1183708227

Reply:
It doesn't matter here but it's easier to know what's going on in the test if we make it explicit.

Reply:
Okay, I'll drop the commit.